src/api.c

```diff
@@ -210,7 +255,7 @@ API int seccomp_reset(scmp_filter_ctx ctx, uint32_t def_action)
 	if (ctx == NULL || db_col_action_valid(NULL, def_action) < 0)
```

Comment:
Argh - github won't let me comment on code outside of the area that was changed.

Do you need a `_rc_filter()` call on line 230 of src/api.c in `seccomp_api_set()`?  It's returning `-EINVAL` without being checked

Reply:
Ha, `seccomp_api_set()` was the other function I caught went I double checked src/api.c :)

Thanks for noticing this, should be fixed in the next force push.